IDLE_DWELL/Find_Idle_Dwells.py

```diff
@@ -0,0 +1,331 @@
+                                                     "inbound_ecs_taken": False,
+                                                     "outbound_ecs_taken": False}
+
+        # EPERIGEE - If this is this line an EPERIGEE line, set the perigee passage
+        #                    eperigee state to True
+        if "EPERIGEE" in working_array[index]["tlmsid_string"]:
+            # Set the state of perigee in the perigee passage dict to True
+            perigee_passage_dict["eperigee_state"] = True
+            
+        # OORMPEN - If this is this line a RADMON ENABLE initialize the
+        #                     initialize the perigee passage dict
+        if "OORMPEN" in working_array[index]["tlmsid_string"]:
+            # Since RADMON is now enabled,  you no longer need the data stored
+            # in the perigee passage dict for this perigee passage So re-init the dict. 
+            perigee_passage_dict = {"radmon_dis_date": "1999:001:00:00:00.00",
+                                                     "radmon_dis_time": apt.secs("1999:001:00:00:00.00"),
+                                                     "radmon_state": "ENAB",
+                                                     "eperigee_state": False,
+                                                     "inbound_ecs_taken": False,
+                                                     "outbound_ecs_taken": False}
+    
+        # SIMTRANS - If this is the SIMTRANS command, process it
+        if working_array[index]["command_type"] == "SIMTRANS":
+            # Fill in the previous obs data structure with values from the
+            # state dictionary
+            
+            # Extract the step value and translate that into an instrument
+            # string
+
+            # Fill in the values that changed due to a SIM translation
+            instrument =  BFCI.Get_Instrument(working_array[index]["tlmsid_string"])
+
+        # START SCIENCE - Is this a start science command? If so you have enough
+        # information to  calculate the dwell time
+        if any( cmd in working_array[index]["tlmsid_string"] for cmd in start_science_command_list):
+
+            # If yes, you have enough information to calculate the Dwell Time
+            dwell_stop_date = working_array[index]["date"].strip()
+            dwell_stop_time = working_array[index]["time"]
+    
+            # Calculate the delta t
+            dwell_time_secs, dwell_time_mins, dwell_time_hours = cd.Calc_Delta_Time(dwell_start_date, dwell_stop_date)
+
+            # Report the dwell time if dwell time is >= to the cutoff, and this is not the gap
+            # between ECS measurements.  Set the report_flag to True.
+            report_flag = True
+
+            # INBOUND ECS- If the present observation is an inbound ECS measurement
+            #                   record that
+            if (perigee_passage_dict["radmon_state"] == "DISA") and \
```

Comment:
This logic is good for the 99% of cases, but heads up that if we are observing a bright source with radmon disabled, this will be mislabeled.